fahrschein-e2e-test/src/test/java/org/zalando/fahrschein/http/apache/ApacheNakadiClientTest.java

```diff
@@ -1,40 +1,17 @@
 package org.zalando.fahrschein.http.apache;
 
-import org.zalando.fahrschein.http.AbstractRequestFactoryTest;
-import org.apache.http.client.config.RequestConfig;
-import org.apache.http.config.ConnectionConfig;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClients;
+import org.zalando.fahrschein.http.AbstractRequestFactoryTest;
+import org.zalando.fahrschein.http.api.ContentEncoding;
 import org.zalando.fahrschein.http.api.RequestFactory;
 
-import java.util.concurrent.TimeUnit;
-
 public class ApacheNakadiClientTest extends AbstractRequestFactoryTest {
 
     @Override
     protected RequestFactory getRequestFactory() {
-        final RequestConfig requestConfig = RequestConfig.custom()
```

Comment:
I simplified the setup of Spring and Apache request factories for the tests, it's not necessarily giving us any benefit.